data/test/index.js

```diff
@@ -12,3 +20,40 @@ describe( 'store', () => {
 		expect( store2.getState() ).toEqual( 'ribs' );
 	} );
 } );
+
+describe( 'select', () => {
+	it( 'registers multiple selectors to the public API', () => {
+		const store = registerReducer( 'reducer1', () => 'state1' );
+		const selector1 = jest.fn( () => 'result1' );
+		const selector2 = jest.fn( () => 'result2' );
+
+		registerSelectors( 'reducer1', {
+			selector1,
+			selector2,
+		} );
+
+		expect( select( 'reducer1', 'selector1' ) ).toEqual( 'result1' );
+		expect( selector1 ).toBeCalledWith( store.getState() );
+
+		expect( select( 'reducer1', 'selector2' ) ).toEqual( 'result2' );
+		expect( selector2 ).toBeCalledWith( store.getState() );
+	} );
+} );
+
+describe( 'query', () => {
+	it( 'passes the relevant data to the component', () => {
+		registerReducer( 'reactReducer', () => 'reactState' );
+		registerSelectors( 'reactReducer', { reactSelector: state => state } );
+		const Component = query( ( selector ) => {
```

Comment:
`selector` is exposed inside `query` as `select`. See:
```js
return mapSelectorsToProps( select, ownProps );
```

Can we keep the convention in tests, too?

Reply:
`select` is already defined above, this just avoids the ESlint issue.

Reply:
Oops, bad Eslint. I hope developers won't follow this test :)